src/node.cc

```diff
@@ -4043,6 +4043,7 @@ static void CheckIfAllowedInEnv(const char* exe, bool is_env,
     // V8 options (define with '_', which allows '-' or '_')
     "--abort_on_uncaught_exception",
     "--max_old_space_size",
+    "--stack_trace_limit"
```

Comment:
Nit: The line above this looks like this list was using trailing commas.

Reply:
thanks, done!